src/xpath.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'xpath',
+    'xpathmodels'
+], function (
+    xpath,
+    makeXPathModels
+) {
+    /* global xpathmodels */
+    /*jshint -W020 */
+    xpathmodels = makeXPathModels();
```

Comment:
Yeah. I couldn't think of a great way for it to not be global due to the way the xpath library is set up.

Would definitely be for cleaning it up so you pass the models to the parser, but not sure how feasible it is at the moment.